src/main/webapp/app/entities/exercise.model.ts

```diff
@@ -75,6 +75,7 @@ export abstract class Exercise implements BaseEntity {
     public maxPoints?: number;
     public bonusPoints?: number;
     public assessmentType?: AssessmentType;
+    public allowComplaintsForAutomaticAssessments?: boolean;
```

Comment:
In the previous state of the programming exercise model the value of allowComplaintsForAutomaticAssessments was set in the constructor. I think you forgot this here.